roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -56,14 +56,35 @@ def test_is_active(is_containerized, group_names, is_active):
     (
         dict(info={
             "Driver": "overlay2",
-            "DriverStatus": []
+            "DriverStatus": [("Backing Filesystem", "xfs")],
         }),
         False,
         [],
     ),
     (
         dict(info={
             "Driver": "overlay",
+            "DriverStatus": [("Backing Filesystem", "btrfs")],
+        }),
+        True,
+        ["'xfs' as the backing storage"],
```

Comment:
because the error message tells you what it's supposed to be as well as what it is 
https://github.com/openshift/openshift-ansible/pull/4757/files#diff-4739a080555d5d9b11dc300197eb46b8R223